examples/meter/meter.js

```diff
@@ -0,0 +1,32 @@
+/** @type {!Element} Meter element. */
+let meterEl;
+/** @type {!Element} Element visually filling the meter. */
+let visualFillEl;
+/** @type {number} Minimum value. */
+let minUsage;
+/** @type {number} Maximum value. */
+let maxUsage;
+/** @type {number} Current value. */
+let usage;
```

Comment:
Are we using `let` and `const` in examples? Also, perhaps these should be locally defined within the functions below rather than defined globally.

Reply:
I think our eslint prevents us from doing so (which I have thoughts about). But not sure if this would impact our build.

Reply:
I'd definitely like to start using es6 syntax and update eslint (lmk if you want help with that), though maybe in a separate PR :D

Reply:
@smhigley,
@ZoeBijl @mcking65 and I are still in the proces of setting up "code review" workflow after the feedback from TPAC meeting in Japan. Thanks for your patience.

Reply:
I'm 100% in agreement about moving on to ES6, but I think we should resolve #1180 before making any changes to code style. All the `let`/`const` keywords should just be replaced with `var` to land this.

Reply:
What’s the reason we don’t allow `let` & `const`?

Reply:
Code is ES5 for IE11 compat AFAIK. Although IE11 mostly supports those with a caveat on `let` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

Reply:
Thanks @nschonni. We can discuss that as part of our support discussion. I think we were all in favour of dropping IE11 support. But no decision has been made yet.